client/snapshot.go

```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+const SnapshotExportContentType = "application/x.snapd.snapshot"
```

Comment:
for consistency with assertions, this should be called ...MediaType. Also a comment would be good:

// SnapshotExportMediaType is a media type used to identify snapshot exports in the API.

Reply:
I'll push a quick patch with that.